src/main/java/seedu/address/storage/JsonAdaptedTag.java

```diff
@@ -7,7 +7,7 @@
 import seedu.address.model.tag.Tag;
 
 /**
- * Jackson-friendly version of {@link Tag}.
```

Comment:
Jackson is the library responsible for the JSON creation, so I think it's okay if we leave it as it is